db/models/reviews-model.js

```diff
@@ -0,0 +1,39 @@
+const db = require('../connection');
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `SELECT reviews.owner, reviews.title, reviews.review_id, reviews.category, reviews.review_img_url, reviews.created_at, reviews.votes, reviews.designer, CAST(COUNT(comments.review_id) AS INT) AS comment_count
+      FROM reviews
+      LEFT JOIN comments
+      ON reviews.review_id = comments.review_id
+      GROUP BY reviews.review_id
+      ORDER BY reviews.created_at DESC;`
+    )
+    .then((res) => {
+      return res.rows;
+    });
+};
+
+exports.fetchReviewById = (review_id) => {
+  if (!Number.isInteger(+review_id)) {
```

Comment:
Line 19: This works and it's very creative but PSQL will throw an error for you in this case! It's expecting review_id to be INT, and if it's not it will throw a massive strop / "22PO2" error code, which will autmatically end up in catch >> next >> handlers >> PSQLhandler >> Client